crates/astria-composer/src/searcher/executor.rs

```diff
@@ -0,0 +1,333 @@
+use std::time::Duration;
+
+use color_eyre::eyre::{
+    self,
+    bail,
+    eyre,
+    Context,
+};
+use ed25519_consensus::SigningKey;
+use humantime::format_duration;
+use proto::native::sequencer::v1alpha1::{
+    Action,
+    SignedTransaction,
+    UnsignedTransaction,
+};
+use secrecy::{
+    ExposeSecret as _,
+    SecretString,
+    Zeroize as _,
+};
+use sequencer_client::{
+    Address,
+    NonceResponse,
+    SequencerClientExt,
+};
+use tokio::sync::{
+    mpsc,
+    watch,
+};
+use tracing::{
+    debug,
+    error,
+    info,
+    instrument,
+    warn,
+};
+
+use crate::Config;
+
+pub(super) type StatusReceiver = watch::Receiver<Status>;
+pub(super) type Sender = mpsc::Sender<Vec<Action>>;
```

Comment:
Would prefer a more specific name for this - `Sender` is too generic - would prefer a `Vec<Action>` be renamed to something like  a `bundle` (just for ex.) and this to be `BundleSender`

Reply:
this follows the naming scheme used in the conductor: https://github.com/astriaorg/astria/blob/937f710d23cdde80a2768309f12105d46e55129b/crates/astria-conductor/src/executor.rs#L45

externally to this module this appears as `executor::Sender`

Reply:
Type aliases should be uses sparsely and only when they improve code clarity. A good example is `type Result<T> = Result<T, CrateSpecificError>`. For things like `Sender<ExecutorCommand>` they don't really improve clarify because a) this type is only used in 2 or so places, and b) I have to look up the alias to understand what's inside. The overhead isn't worth it. We should change conductor accordingly.